<html>

    <head>
  <meta charset="utf-8">
  <title>Normalization and expression changes in predefined sets of proteins using 2D gel electrophoresis: a proteomic study of L-DOPA induced dyskinesia in an animal model of Parkinsons disease using DIGE | Research Group at Uppsala University</title>
  <meta content="width=device-width, initial-scale=1" name="viewport">
  <meta content="Webflow" name="generator">
  <link href="/css/normalize.css" rel="stylesheet" type="text/css">
  <link href="/css/webflow.css" rel="stylesheet" type="text/css">
  <link href="/css/caramba.webflow.css?1710165300" rel="stylesheet" type="text/css">
  <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js"></script>
  <script type="text/javascript">
    WebFont.load({
      google: {
        families: ["Ubuntu:300,300italic,400,400italic,500,500italic,700,700italic","Raleway:regular,500,600,700,800,900"]
      }
    });
  </script>
  <script src="/js/modernizr.js" type="text/javascript"></script>
  <link href="/img/logos/caramba_header_logo.png" rel="shortcut icon" type="image/x-icon">
  <link href="/img/logos/caramba_header_logo.png" rel="apple-touch-icon">
  <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-86146183-1', 'auto');
    ga('send', 'pageview');
  </script>
</head>


    
  <div class="section-header" id="section-header">
    <div class="section-header-container w-container">
      <div class="navbar w-nav" data-animation="default" data-collapse="medium" data-contain="1" data-doc-height="1" data-duration="400" data-no-scroll="1">
        <div class="navbar-container w-container">
          <a class="navbar-brand w-nav-brand" href="/">
              <img class="navbar-logo" src="/img/logos/rect205.png" height="50">
          </a>
          <div class="navbar-menubutton w-nav-button">
            <div class="w-icon-nav-menu"></div>
          </div>
          <a class="navbar-uulogo-link" href="https://www.medsci.uu.se/caramba/" target="_blank"><img class="navbar-uulogo-img" src="/img/logos/uulogo_64x64.png"></a>
          <nav class="navbar-menu w-nav-menu" role="navigation" style="float: right">
              
              <a class="navbar-link w-nav-link" href="/project/">Projects</a>
              
              <a class="navbar-link w-nav-link" href="/people/">People</a>
              
              <a class="navbar-link w-nav-link" href="/facility/">Facility</a>
              
              <a class="navbar-link w-nav-link" href="/publication/">Publications</a>
              
              
              
              <a class="navbar-link w-nav-link" href="/education/">Education</a>
              
              <a class="navbar-link w-nav-link" href="/about/">About</a>
              
          </nav>
        </div>
      </div>
    </div>
  </div>

              <div>
          <div class="section-content-container w-container content">

        <h1 style="font-size: 1.4em">Normalization and expression changes in predefined sets of proteins using 2D gel electrophoresis: a proteomic study of L-DOPA induced dyskinesia in an animal model of Parkinsons disease using DIGE</h1>
        <p><a href="/publication">&larr; Back to publications</a></p>

        <p>Published: 2006-10-28</p>

        <h4>Formatted citation</h4>
        <p>
            Kultima K, Scholz B, Alm H, Sköld K, Svensson M, Crossman AR, Bezard E, Andrén PE, Lönnstedt I..
            <a href="http://dx.doi.org/10.1186/1471-2105-7-475" target="_blank">Normalization and expression changes in predefined sets of proteins using 2D gel electrophoresis: a proteomic study of L-DOPA induced dyskinesia in an animal model of Parkinsons disease using DIGE</a>.
            <br><em>BMC Bioinformatics</em>.
             (2006).
            DOI: <a href="http://dx.doi.org/10.1186/1471-2105-7-475" target="_blank">10.1186/1471-2105-7-475</a>
            
        </p>
        <h4>Abstract</h4>
        <p>BACKGROUND: Two-Dimensional Difference In Gel Electrophoresis (2D-DIGE) is a powerful tool for measuring differences in protein expression between samples or conditions. However, to remove systematic variability within and between gels the data has to be normalized. In this study we examined the ability of four existing and four novel normalization methods to remove systematic bias in data produced with 2D-DIGE. We also propose a modification of an existing method where the statistical framework determines whether a set of proteins shows an association with the predefined phenotypes of interest. This method was applied to our data generated from a monkey model (Macaca fascicularis) of Parkinsons disease. RESULTS: Using 2D-DIGE we analysed the protein content of the striatum from 6 control and 21 MPTP-treated monkeys, with or without de novo or long-term L-DOPA administration. There was an intensity and spatial bias in the data of all the gels examined in this study. Only two of the eight normalization methods evaluated (2D loess&#43;scale and SC-2D&#43;quantile) successfully removed both the intensity and spatial bias. In SC-2D&#43;quantile we extended the commonly used loess normalization method against dye bias in two-channel microarray systems to suit systems with three or more channels.Further, by using the proposed method, Differential Expression in Predefined Proteins Sets (DEPPS), several sets of proteins associated with the priming effects of L-DOPA in the striatum in parkinsonian animals were identified. Three of these sets are proteins involved in energy metabolism and one set involved proteins which are part of the microtubule cytoskeleton. CONCLUSION: Comparison of the different methods leads to a series of methodological recommendations for the normalization and the analysis of data, depending on the experimental design. Due to the nature of 2D-DIGE data we recommend that the p-values obtained in significance tests should be used as rankings only. Individual proteins may be interesting as such, but by studying sets of proteins the interpretation of the results are probably more accurate and biologically informative.</p>

            </div>
</div>

      <div class="section-footer">
    <div class="section-footer-container w-container">
        
        <div class="footer-text">
            <p>
            <strong>CARAMBA Research Group</strong><br>
            Department of Medical Sciences<br>
            Uppsala University, Sweden<br>
            GitHub: <a href="https://github.com/caramba-uu">github.com/caramba</a>
            </p>
        </div>
    </div>
  </div>

  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js" type="text/javascript"></script>
  <script src="/js/webflow.js" type="text/javascript"></script>
  
</body>
</html>

